Extensions/xkit_patches.js

```diff
@@ -922,27 +922,15 @@ XKit.extensions.xkit_patches = new Object({
 				 * @return {Array<Object>} The posts
 				 */
 				get_posts: async function(without_tag, can_edit) {
-					var posts = [];
+					const selector = `[data-id]${(without_tag != null ? `:not(.${without_tag})` : "")}`;
```

Comment:
if an argument isn't passed it will be `undefined`, which *does* loosely equal `null`, but it's clearer to just compare strictly with undefined. this double template is a bit rough too...

```suggestion
					let selector = "[data-id]";

					if (without_tag !== undefined) {
						selector += `:not(.${without_tag})`;
					}
```